Dockerfile

```
@@ -0,0 +1,72 @@
+FROM nvidia/cuda:11.8.0-devel-ubuntu22.04 AS dev
```

```suggestion
ARG MAX_JOBS=4

FROM nvidia/cuda:11.8.0-devel-ubuntu22.04 AS dev
```

Forward declare the argument and set sensible default